packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -136,6 +141,11 @@ new appmesh.VirtualGateway(stack, 'gateway2', {
     healthCheck: {
       interval: cdk.Duration.seconds(10),
     },
+    tlsCertificate: appmesh.TlsCertificate.file({
```

Comment:
This checks the same `file` method twice. Can we use the `acm` method here instead?

Reply:
We thought the additional test coverage (our API is only validating the ARN) didn't justify the time and complexity of creating a CA. Thoughts?

Reply:
Is there any additional complexity to creating the CA in this test other than an extra resource in the Stack?

Reply:
no the concern was just adding that extra resource. (typo above I was talking an ACM certificate not a Certificate Authority)

Reply:
OK, if it doesn't add any complexity beyond that one resource, I think it's worth it to have it in the tests.